src/Datadog.Trace/ClrProfiler/InstrumentationDefinitions.cs

```diff
@@ -0,0 +1,234 @@
+// <copyright file="InstrumentationDefinitions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler
+{
+    internal static class InstrumentationDefinitions
+    {
+        internal static NativeCallTargetDefinition[] GetAllDefinitions()
+        {
+            var defsArray = new NativeCallTargetDefinition[218];
+            defsArray[0] = new NativeCallTargetDefinition("AerospikeClient", "Aerospike.Client.AsyncCommand", "ExecuteCommand",  new string[] { "System.Void" }, 4, 0, 0, 4, 65535, 65535, "Datadog.Trace, Version=1.28.6.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb", "Datadog.Trace.ClrProfiler.AutoInstrumentation.Aerospike.AsyncCommandIntegration");
```

Comment:
Non-blocking, but each line here can be 
```
defsArray[0] = new(....);
```

or we could go further and use
```
var defsArray = new NativeCallTargetDefinition
{
    new(....),
    new(....),
    new(....),
    // ...
};

Reply:
done in 73f5a81